xcube/core/maskset.py

```diff
@@ -177,6 +191,35 @@ def get_mask(self, flag_name: str):
         self._masks[flag_name] = mask_var
         return mask_var
 
+    def get_cmap(self, default: str = "viridis") -> matplotlib.colors.Colormap:
+        """Get a suitable color mapping for use with matplotlib.
+
+        Args:
+            default: Default color map name in case a color mapping
+                cannot be created, e.g., ``flag_values`` are not defined.
+
+        Returns:
+            An suitable instance of ```matplotlib.colors.Colormap```
```

Comment:
```suggestion
            A suitable instance of ```matplotlib.colors.Colormap```
```